extensions/blocks/podcast-player/style.scss

```diff
@@ -345,7 +366,8 @@ $player-background: transparent;
 	.mejs-container,
 	.mejs-embed,
 	.mejs-embed body,
-	.mejs-container .mejs-controls {
+	.mejs-container .mejs-controls,
+	.mejs-mediaelement {
```

Comment:
Minor: Just noticed this because a selector was added. I'm not entirely certain what the code convention in Jetpack is around ordering selectors but this does seem a bit out of order. It might be worth double-checking. If anything I'd suggested we at least move the container-related selectors next to each other.

```
.mejs-container,
.mejs-container .mejs-controls,
```

Reply:
Addressed in the follow-up cleanup PR: https://github.com/Automattic/jetpack/pull/15512/commits/85d6b0cf267ed3d7baaadd4a9b64c798a0fb9e04

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/15512/commits/85d6b0cf267ed3d7baaadd4a9b64c798a0fb9e04